src/browser/queries/index.ts

```diff
@@ -0,0 +1,265 @@
+import path from "path";
+import fs from "fs";
+import {
+    Matcher,
+    MatcherOptions,
+    queries as baseQueries,
+    waitForOptions as WaitForOptions,
+} from "@testing-library/dom";
+
+import { BaseWithExecute, BrowserBase, ElementBase } from "./wdio-types.js";
+import {
+    QueryArg,
+    Config,
+    QueryName,
+    WebdriverIOQueries,
+    WebdriverIOQueriesChainable,
+    ObjectQueryArg,
+    SerializedObject,
+    SerializedArg,
+} from "./types.js";
+
+declare global {
+    interface Window {
+        TestingLibraryDom: typeof baseQueries & {
+            configure: typeof configure;
+        };
+    }
+}
+
+/*
+eslint-disable
+@typescript-eslint/explicit-function-return-type
+*/
+
+const DOM_TESTING_LIBRARY_UMD_PATH = path.join(
```

Comment:
since this is a complete copy of https://github.com/testing-library/webdriverio-testing-library, i think we should give them credit, as required by MIT license

Reply:
copyright advertise added on top of index.ts